### ISSUE TITLE
[Bug]:No matter which label I search for, the first label is displayed

### ISSUE THREAD

**Issue by wu4339** [federicoiosue/Omni-Notes#971]

**Comment by federicoiosue** [comment]
Yeah, I'm experiencing the same issue, but I've not been able to replicate nor on emulator or on other devices than on mine.
It may depend on the number on notes/tags too: would you mind sharing a screenshot of Settings->Statistics to help me understanding if this could impact or not someway in your case too?

**Comment by wu4339** [comment]
> Yeah, I'm experiencing the same issue, but I've not been able to replicate nor on emulator or on other devices than on mine.
> It may depend on the number on notes/tags too: would you mind sharing a screenshot of Settings->Statistics to help me understanding if this could impact or not someway in your case too?

![Screenshot_2024-04-03-09-37-59-462_it.feio.android.omninotes-edit.jpg](https://github.com/federicoiosue/Omni-Notes/assets/16701668/4fb2ea06-255a-47e6-8930-437bb8ce4f0f)
